### PR TITLE
update docs: add pageOutOfRange option

### PR DESCRIPTION
**Pull request by daria-sieroshtan**
Updating docs to reflect [PR#260](https://github.com/KnpLabs/knp-components/pull/260) changes.

### CODE REVIEW

File: docs/pager/config.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The list of existing options are:
 | filterFieldWhitelist       | array          |               | FiltrationSubscriber                            |
 | filterFieldParameterName   | string         | filterParam   | FiltrationSubscriber                            |
 | filterValueParameterName   | string         | filterValue   | FiltrationSubscriber                            |
+| pageOutOfRange             | string         | ignore        | PaginationSubscriber                            |
```
Comment (daria-sieroshtan) [inline docs/pager/config.md:26]: I am not sure about list of subscribers.

Comment (garak) [reply docs/pager/config.md:26]: What you're not sure about?

Comment (daria-sieroshtan) [reply docs/pager/config.md:26]: Whether I am not missing anything.

Comment (garak) [reply docs/pager/config.md:26]: Well, I have to admin that last column is not super-clear to me. For example, I don't get why `pageParameterName` should be for SortableSubscriber only...
I think we can go with PaginationSubscriber

**Pull request merged**